scripts/run_integration_tests.sh

```diff
@@ -40,46 +42,67 @@ else
     echo "Not building layers, ensure they've already been built or re-run with 'BUILD_LAYERS=true DD_API_KEY=XXXX ./scripts/run_integration_tests.sh'"
 fi
 
+# Add local build to node_modules so `serverless-plugin.yml` also has access to local build.
+yarn build
```

Comment:
I think we need a yarn install here. When I first ran the tests I got an error, yarn install fixed it.